test_calc.py

```diff
@@ -0,0 +1,28 @@
+"""
+Unit tests for calc.py
+"""
+
+import unittest
+from calc import calculate
+
+class TestCalculator(unittest.TestCase):
```

Comment:
You should cover also different input formats like:
2
2.0
2e-3
etc.

Remember: the code should adhere to the DRY and KISS programming principles, so use pytest.mark.param or hypothesis framework if needed.